src/modules/esl-base-popup/core/esl-base-popup.ts

```diff
@@ -43,38 +40,36 @@ export class ESLBasePopup extends ESLBaseElement {
   @boolAttr() public closeOnEsc: boolean;
   @boolAttr() public closeOnOutsideAction: boolean;
 
-  @jsonAttr<PopupActionParams>({defaultValue: {silent: true, force: true, initiator: 'init'}})
```

Comment:
We can remove `silent` from the default object literal in case it is false.

Reply:
Fixed. Thanks!